libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -247,7 +247,11 @@ export const [hasActiveLink, setActiveLink, getActiveLink] = (() => {
 
       [`${origin}${pathname}`, pathname].forEach((path) => {
```

Comment:
With this new approach, I'd test a bit more to justify the necessity of the `pathname` check. Say we have the following element: `<a href="/path/to/page">Page</a>`. We needed the `pathname` check for the query selector, which would have been `a[href = '/path/to/page']`. So we were checking an attribute value, which could have been either a full URL or just a path. Moving to the `el.href` approach, `pathname` might be irrelevant, as its result would always be a full URL.

Reply:
Yes, I agree. I will make the necessary change and update the PR.